components/SearchBar/SearchBar.jsx

```diff
@@ -93,11 +94,45 @@ class SearchBar extends Component {
 
   onKeyUp(evt) {
     const eventKey = evt.keyCode
+    evt.stopPropagation()
+    evt.preventDefault()
     // if return is pressed
     if (eventKey === 13) {
       this.setState({ searchState: 'filled' }, function() {
         this.search()
       })
+    } else if (eventKey === 39) { // right arrow key is pressed
+      const suggestion = this.state.suggestions.length > 0 ? this.state.suggestions[0] : null
```

Comment:
I guess yes because that would assign `undefined` if suggestions array does not have any element in it and `if (suggestion)` would not evaluate to true in both cases, `undefined` and `null`